packages/config/src/memory/memory.config.ts

```diff
@@ -140,8 +181,15 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
       removeUndefined(existing);
       this.put(existing);
     }
-    // TODO this overwrites existing layers
-    existing.layers[0][i.projection] = i.id;
+    // The latest imagery overwrite the earlier ones.
+    const existingImageryId = existing.layers[0][i.projection];
+    if (existingImageryId) {
+      existing.layers[0][i.projection] = findLatestId(i.id, existingImageryId);
+      this.duplicateImagery.push(existing);
```

Comment:
NIT: this wont actually give you useful information, it will overwrite the old ID so you only get that there is a duplicate not what the IDs actually are.

Reply:
Do you think which id should be useful to catch. Just the old ids that been overwrited?
I will merge this first and fix in https://github.com/linz/basemaps/pull/2812.